### DISCUSSION TITLE
How to put conda env before prompt

### DISCUSSION THREAD

**Discussion by lminer** [starship/starship#4979] [Q&A]
I'm trying to figure out how to mimic pure prompt when it comes to the positioning of the conda environment/python virtual env. Basically, I'd like to move it from above the prompt, to before it so something like:

```
~
myenv>
```

**Comment by mickimnet** [comment]
You can do that by creating your own configuration and changing the order in the `format` section (see: [Configuration/Default Prompt Format](https://starship.rs/config/#default-prompt-format)).

You can create a basic configuration file with `starship print-config --default >~/.config/starship.toml` or you can use a preset as a basis with `starship preset $Preset_Name >~/.config/starship.toml`. You can get the preset names with `starship preset --list`.

Hope that helps.
